scripts/perf/README.md

```diff
@@ -0,0 +1,18 @@
+# Perf runner
```

Comment:
```suggestion
# Performance Runner
```
Can you add a little blurb under this heading about what the perf runner actually does? What files will be generated when you run these scripts? How should someone view these files? Will this graph change based on the machine the script is run on? It might also be helpful to have a section on how to read the flamegraph, like, what do the x and y axes represent? The guide as it stands now is pretty sparse.

Reply:
I'd prefer to do that separately from this PR. I'm mostly wanting to get this merged soon so we can start getting value from it asap.

Reply:
Alright, it's fine if we do it later, but can you please make an issue for this? Since this isn't really tied to any particular project we're working on, I could see us forgetting about documentation altogether.

Reply:
https://github.com/aws/s2n-tls/issues/2957